test/integration/app-definition-integration.js

```diff
@@ -0,0 +1,57 @@
+import { expect } from 'chai'
+import { before, describe, test, after } from 'mocha'
+import { client } from '../helpers'
+
+describe('AppDefinition api', function () {
+  let organization
+
+  before(async () => {
+    organization = await client()
+      .getOrganizations()
+      .then((response) => response.items[0])
+  })
+
+  after(async () => {
+    organization.getAppDefinitions().then((response) => {
+      response.items.map((appDefinition) => appDefinition.delete())
+    })
```

Comment:
You need to use `await Promise.all(...)` here. Or do this:

```suggestion
    const appDefinitions = await organization.getAppDefinitions();
    for (const appDefinition of appDefinitions) {
      await appDefinition.delete()
    }
```